sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -1114,6 +1114,56 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates Provincial Daily Disbursement Report file processing notifications.
+   * Currently multiple notifications are created as integration contacts
+   * can be multiple.
+   * @param attachmentFileContent file content for the daily disbursement report file.
+   * @param fileName file name for the daily disbursement report file.
+   */
+  async saveProvincialDailyDisbursementReportProcessingNotification(
+    attachmentFileContent: string,
+    fileName: string,
+  ): Promise<void> {
+    const auditUser = this.systemUsersService.systemUser;
+    const { templateId, emailContacts } =
+      await this.assertNotificationMessageDetails(
+        NotificationMessageType.MinistryNotificationProvincialDailyDisbursementReceipt,
+      );
+    if (!emailContacts?.length) {
+      return;
+    }
+    const provincialDailyDisbursementReportProcessingNotifications = [];
```

Comment:
Can we follow the same approach from other similar methods and use a `map` instead of the `for`?
```ts
const ministryNotificationsToSend = emailContacts.map((emailContact) => ({
```